protocol/chainlib/chainproxy/rpcInterfaceMessages/restMessage_test.go

```diff
@@ -7,14 +7,15 @@ import (
 func TestRestMessage(t *testing.T) {
 	// Test GetParams method
 	restMessage := RestMessage{
-		Path: "eth_getTransactionByHash",
-		Msg:  []byte{1, 2, 3, 4, 5},
+		Path:     "eth_getTransactionByHash",
+		Msg:      []byte{1, 2, 3, 4, 5},
+		SpecPath: "eth_getTransactionByHash",
 	}
 
 	// Test GetParams method
 	params := restMessage.GetParams()
-	if params != nil {
-		t.Errorf("Expected nil, but got %v", params)
+	if params == nil {
```

Comment:
you dont need to do this check you can do 
require.Nil(t, params)

Reply:
no need for print as well the require is enough.

Reply:
fixed in : fd83ad126719d12e3ec38756fd82b04c61f4af1d